lib/api/multiObjectDelete.js

```diff
@@ -492,8 +493,11 @@ function multiObjectDelete(authInfo, request, log, callback) {
         function parseXML(next) {
             return _parseXml(request.post,
                 (err, quietSetting, objects) => {
-                    if (err || objects.length < 1 || objects.length > 1000) {
-                        return next(errors.MalformedXML);
+                    if (err || objects.length < 1 || objects.length > constants.maximumBatchDeleteObjects) {
+                        return next(errors.MalformedXML.customizeDescription(
+                            `The provided request tries to delete ${objects.length} objects, ` +
```

Comment:
is this the message that AWS returns? (the `AWS standard` part looks weird)

as a general rule of thumb, we want to stick to same behavior (esp errors) as AWS